### PR TITLE
Updated naming conventions for CoDICE

### PR DESCRIPTION
**Pull request by bourque**
# Change Summary

## Overview
This PR updates the Naming Conventions docs to include the expected instrument and descriptor names for CoDICE, as well as an updated example.

## Updated Files

- `docs/source/development-guide/style-guide/naming-conventions.rst`

### CODE REVIEW

**Review by greglucas — APPROVED** [review]
I think this makes sense, but just to verify, I thought the codice team wanted `hi` in the `<descriptor>` field instead of the instrument field?

**Comment by bourque** [comment]
I will ask about that when I meet with the CoDICE folks tomorrow. These updates reflect what is defined in the latest version of our algorithm document, but perhaps they want to change it.

File: docs/source/development-guide/style-guide/naming-conventions.rst [hunk 1]
```diff
@@ -14,8 +14,8 @@ The general filename convention is as follows::
     imap_<instrument>_<datalevel>_<descriptor>_<startdate>_<enddate>_<version>.<extension>
 
 * ``<instrument>`` is the IMAP instrument associated with the data product. Acceptable values include: ``codice``,
-  ``glows``, ``hi-45``, ``hi-90``, ``hit``, ``idex``, ``lo``, ``mag``, ``swapi``, ``swe``, ``ultra-45``, and
-  ``ultra-90``
+  ``codice-hi``, ``codice-lo``, ``glows``, ``hi-45``, ``hi-90``, ``hit``, ``idex``, ``lo``, ``mag``, ``swapi``, ``swe``,
```
Comment (maxinelasp) [inline docs/source/development-guide/style-guide/naming-conventions.rst]: From meeting: Should be one instrument "codice" with further distinctions in the descriptor

File: docs/source/development-guide/style-guide/naming-conventions.rst [hunk 2]
```diff
@@ -35,7 +35,7 @@ The general filename convention is as follows::
   ``burst-7`` for 'burst' mode and the 7th repointing). This field can have any text or numbers as long as it doesn't
   include underscores. The following is a list of expected descriptors for each instrument:
 
-  * CoDICE: ``hi``, ``lo``
+  * CoDICE: ``priority``, ``sw-angular``, ``nsw-angular``, ``hskp``, ``eng``, ``pha``, ``sw-species``, ``nsw-species``, ``omni``, ``sectored``
```
Comment (maxinelasp) [inline docs/source/development-guide/style-guide/naming-conventions.rst]: Products with "lo" or "hi": omni,

**Review by maxinelasp — CHANGES_REQUESTED** [review]
instrument should be only codice

**Comment by bourque** [comment]
Putting this in draft / on hold until I hear back from the CoDICE folks -- I sent around an email about this with Joey, Micheal, and Mihir in attempt to finalize all of this.

**Comment by bourque** [comment]
Updated this with new instrument name and descriptors based on feedback from Joey. This is ready for review/merge now.

**Review by greglucas — APPROVED** [review]

**Review by tech3371 — APPROVED** [review]

**Pull request merged**